minter-suite/minter-suite-supplemental.md

```diff
@@ -139,3 +139,49 @@ console.log(
   merkleProof
 );
 ```
+
+---
+
+## Pre-Assigning Token Hash Seeds
+
+The shared randomizer `SharedRandomizerV0` enables an artist to pre-assign token hash seeds for tokens on a project. This allows an artist to assign a specific hash seed to a token before it is minted, and then mint the token with that hash seed at a later time.
+
+The process for pre-assigning token hash seeds is as follows:
+
+1. Identify and confirm that your contract is using a `SharedRandomizerV0` contract. Visit your core contract on etherscan (or appropriate L2 explorer) and inspect the read-only function `randomizerContract()` to confirm that the randomizer contract is a `SharedRandomizerV0`.
+
+1. Navigate to the `SharedRandomizerV0` contract on etherscan (or appropriate L2 explorer), and connect your artist wallet. Alternatively, if your artist wallet is a multisig, interact with the `sharedRandomizerV0` in something like Gnosis Safe's Transaction Builder (this is more efficient and convenient when assigning hash seeds for many tokens).
```

Comment:
`(or appropriate L2 explorer)` -> `(or appropriate L2 explorer, eg. basescan)`

Reply:
updated in [977abdc](https://github.com/ArtBlocks/artblocks-docs/pull/312/commits/977abdc26fbfcd51f847cc118c13ce7bac59b0fa)